playground/mocks/data/idp/idx/safe-mode-enduser-enrollment.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
file name should be something like `safe-mode-credential-enrollment-intent.json` or so. SIW doesn't really care much about end user settings page, it care about what comes in the response. 
And well, this is similar to a required enrollment, in the sense that we have no way to move forward, but instead we need to cancel. This response does not have a cancel though... so another suggestion can be `safe-mode-credential-enrollment-no-cancel-no-skip.json`

Reply:
Yea, you have a good point. I named this to test the flow when coming from end user with no cancel and no skip links. So probably `safe-mode-credential-enrollment-no-cancel-no-skip.json` is a good option

Reply:
Talked to Mau offline. Naming according to intent sounds good 👍